Source/Private/Write-PassThruOutput.ps1

```diff
@@ -0,0 +1,10 @@
+function Write-PassThruOuput {
+    [CmdletBinding()]
+    param (
+        [String[]]
+        $Message
+    )
+    foreach ($line in $($Message)) {
+        Write-Information $line
```

Comment:
Curious, would this produce any output to console or host or anywhere?

Reply:
Will this parameter help? `-InformationAction Continue`?

Reply:
Yes, you are exactly right, this is needed to make sure the output will always be written as expected.